R/derive_param_tte.R

```diff
@@ -91,13 +91,15 @@
 #'   single dataset.
 #'
 #'   \item For each patient the first observation (with respect to the `ADT`
-#'   variable) from the single dataset is selected. }
+#'   variable) from the single dataset is selected. If there is more than one
+#'   event with the same date, the first event with respect to the order of
```

Comment:
does time ever need to be considered for these types of events?

Reply:
not really, TTE analyses at the lowest unit is calculated in days but often it could even be in months. You could use ADTM within the source objects for event/censoring if you wanted to include time in the initial sorting, but after that you just need to know a date and where it came from.

Reply:
If you set `create_datetime = TRUE`, time is considered. I have updated the documentation to be more precise.

However, as @rossfarrugia pointed out, this is a very unlikely scenario. I've never seen a TTE analysis where time is considered.

Reply:
thanks both for explanation!